src/metabase/models/user.clj

```diff
@@ -98,7 +100,14 @@
              membership-exists?)
         (db/simple-delete! PermissionsGroupMembership
           :group_id (u/the-id (perms-group/admin))
-          :user_id  id))))
+          :user_id  id)))
+    (when (and still-superuser?
+               (false? active?)
+               (not (db/exists? PermissionsGroupMembership
+                      :group_id (:id (perms-group/admin))
+                      :user_id [:!= id])))
+      (throw (ex-info (str perms-group-membership/fail-to-remove-last-admin-msg)
+                      {:status-code 400}))))
```

Comment:
I don't love the nearly-duplicated logic between `user/pre-update` and `perms-group-membership/pre-delete`, but it was just different enough that I thought DRYing it up would be counter-productive.